src/dotnet/commands/dotnet-migrate/Program.cs

```diff
@@ -80,13 +80,8 @@ public static int Run(string[] args)
             }
             catch (Exception ex)
```

Comment:
I would be a bit more comfortable with this change if we were catching some more specific exceptions before this one.